src/main/java/com/amazonaws/secretsmanager/caching/SecretCacheConfiguration.java

```diff
@@ -237,4 +250,45 @@ public SecretCacheConfiguration withVersionStage(String versionStage) {
         return this;
     }
 
+    /**
+     * Returns the refresh jitter that is used when force refreshing secrets.
+     *
+     * @return The maximum jitter sleep time in milliseconds used with refreshing secrets.
+     */
+    public long getForceRefreshJitter() {
```

Comment:
Nit: you could call this field forceRefreshJitterMillis, that's generally convention in Java.